etc/adminhtml/system.xml

```diff
@@ -1,102 +1,84 @@
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+	<system>
+        <section id="yotpo" translate="label" sortOrder="10" showInDefault="1" showInWebsite="1" showInStore="1">
             <class>separator-top</class>
-            <label>Yotpo Product Reviews Software</label>
+            <label>Reviews &amp; Visual Marketing</label>
             <tab>yotpo</tab>
-            <resource>Yotpo_Yotpo::settings</resource>
-            <group id="settings" translate="label" type="text" sortOrder="10" showInDefault="1" showInWebsite="0" showInStore="1">
-                <label>Yotpo Widget Configurations</label>
-                                        <comment>
-                <![CDATA[
-                    <p>Please enter in your <font color="red">Api App Key</font>  and <font color="red">Secret</font> in the fields below to finish the installation.</p>
-                        <p>If you need help the Yotpo support team is here 24/7, don't hesitate to 
-                            <a href="https://yotposupport.zendesk.com/account/dropboxes/20158038#/dropbox/suggestions/ask" target="_blank" title="yotpo-support">contact us.</a>
-                              </p>
-                    <p>Click <a href="http://support.yotpo.com/entries/22635391-advanced-magento-features" target="_blank" 
-                      title="yotpo">here</a> for the advanced customization guide. 
-                    </p><br/>
-                    <p style="font-size:18px;">Dont have an App Key? <a style="font-size:18px;" href="https://my.yotpo.com/register?utm_source=customers_magento_admin&utm_medium=link&utm_campaign=magento_admin_signin_link" target="_blank" 
-                      title="yotpo">Click here to sign up / sign in</a></p>
-                ]]>
-            </comment>
-                <field id="app_key" translate="label" type="text" sortOrder="1" showInDefault="1" showInWebsite="0" showInStore="1">
-                    <label>App key</label>
-                    <validate>required-entry</validate>
-                    <backend_model>Yotpo\Yotpo\Model\Config\Backend\ValidateSettings</backend_model>
-                    <comment><![CDATA[Get your appkey at your yotpo account setting page. Click
-                        <a href="https://my.yotpo.com/register?utm_source=customers_magento_admin&utm_medium=link&utm_campaign=magento_admin_signin_link" target="_blank" title="yotpo-sign-up">here</a>
-                        to sign in to yotpo]]>
-                    </comment>                    
+            <resource>Yotpo_Yotpo::config</resource>
+            <group id="settings" translate="label" type="text" sortOrder="10" showInDefault="1" showInWebsite="1" showInStore="1">
+                <label><![CDATA[Yotpo Widget Configurations <div style="float:right;margin:0 20px;font-weight:normal;line-height:23px;font-size:1.2rem;">ver. 2.7.5</div>]]></label>
```

Comment:
In theory it's possible to create a custom block that will be loaded here with a backend class attached to it, but it would be much easier to just search & replace the version number when it's updated.
If you want to do that please contact our PM & we'll estimate & schedule.